src/nupic/os/Regex.cpp

```diff
@@ -46,7 +47,14 @@ namespace nupic
       exactRegExp += re;
       if (re[re.length()-1] != '$') 
         exactRegExp += '$';
-      
+
+#if defined(NTA_OS_WINDOWS)
```

Comment:
Yes originally PCRE was used for regex. With the adoption of C++11 we could use this, and remove the additional Windows library. This was also tested on Travis, and Darwin can use this too. GCC 4.8 had issues; https://travis-ci.org/rcrowder/nupic.core/builds/40795605